core/src/main/java/bisq/core/grpc/CoreApi.java

```diff
@@ -48,35 +44,95 @@
-                   OpenOfferManager openOfferManager,
-                   User user) {
+                   TradeStatisticsManager tradeStatisticsManager) {
+        this.coreOffersService = coreOffersService;
         this.paymentAccountsService = paymentAccountsService;
         this.walletsService = walletsService;
-        this.offerBookService = offerBookService;
         this.tradeStatisticsManager = tradeStatisticsManager;
-        this.createOfferService = createOfferService;
-        this.openOfferManager = openOfferManager;
-        this.user = user;
     }
 
     public String getVersion() {
         return Version.VERSION;
     }
 
+    ///////////////////////////////////////////////////////////////////////////////////////////
+    // Offers
+    ///////////////////////////////////////////////////////////////////////////////////////////
+
+    public List<Offer> getOffers(String direction, String fiatCurrencyCode) {
+        return coreOffersService.getOffers(direction, fiatCurrencyCode);
+    }
+
+    public void createOffer(String currencyCode,
+                            String directionAsString,
+                            long priceAsLong,
+                            boolean useMarketBasedPrice,
+                            double marketPriceMargin,
+                            long amountAsLong,
+                            long minAmountAsLong,
+                            double buyerSecurityDeposit,
+                            String paymentAccountId,
+                            TransactionResultHandler resultHandler) {
+        coreOffersService.createOffer(currencyCode,
+                directionAsString,
+                priceAsLong,
+                useMarketBasedPrice,
+                marketPriceMargin,
+                amountAsLong,
+                minAmountAsLong,
+                buyerSecurityDeposit,
+                paymentAccountId,
+                resultHandler);
+    }
+
+    public void createOffer(String offerId,
+                            String currencyCode,
+                            OfferPayload.Direction direction,
+                            Price price,
+                            boolean useMarketBasedPrice,
+                            double marketPriceMargin,
+                            Coin amount,
+                            Coin minAmount,
+                            double buyerSecurityDeposit,
+                            PaymentAccount paymentAccount,
+                            boolean useSavingsWallet,
+                            TransactionResultHandler resultHandler) {
+        coreOffersService.createOffer(offerId,
+                currencyCode,
+                direction,
+                price,
+                useMarketBasedPrice,
+                marketPriceMargin,
+                amount,
+                minAmount,
+                buyerSecurityDeposit,
+                paymentAccount,
+                useSavingsWallet,
+                resultHandler);
+    }
```

Comment:
It took me a minute to realise that the difference between these methods is that one of them takes primitives and the other takes higher level abstractions. I'm leaving this comment as an annotation for myself.